Makefile

```diff
@@ -28,8 +28,8 @@ lint:
 	golangci-lint run -c lint-rule.yaml --timeout=2m
 
 .PHONY: test
-test:
-	PATH=$(shell pwd):${PATH} go test -coverprofile coverage.out -timeout 28m ./...
+test: build
```

Comment:
I'm afraid `build` target is not enough for `test` target.
You have to `install` the binary to the `PATH`.

Reply:
It work, command `PATH=$(shell pwd):${PATH}` add current directory to `PATH` environment variable

Reply:
My fault, not watching carefully.